receiver/m365receiver/README.md

```diff
@@ -1,8 +1,15 @@
-# Microsoft Office 365 Receiver
-Receives metrics from [Microsoft Office 365](https://www.microsoft365.com/)
+# Microsoft 365 Receiver
+Receives metrics from [Microsoft 365](https://www.microsoft365.com/)
 via the [Microsoft Graph API](https://learn.microsoft.com/en-us/graph/api/overview?view=graph-rest-1.0&preserve-view=true),
 and logs via the [Microsoft Management API](https://learn.microsoft.com/en-us/office/office-365-management-api/office-365-management-activity-api-schema).
 
+## Important Notes
+Due to the nature of the APIs being used to collect data from Microsoft, there are some important notes that should be acknowledged before continuing.
+
+- Metrics collected by this receiver can be 48 hours old. This is because the Reports endpoint of the Microsoft Graph API can take up to 48 hours to be updated with new data. The timestamps for metrics will represent when the data was scraped from the API rather than when the data points occurred (48 hours earlier). Because of this, the metrics receiver has a strict scrape interval of 1 hour that cannot be changed. This is done so that the receiver can scrape data from the API once it becomes available, within a reasonable amount of time. This also means that most data points are duplicates. Please keep this in mind when viewing metric data. For more info please see Microsoft's [documentation](https://learn.microsoft.com/en-us/microsoft-365/admin/activity-reports/activity-reports?view=o365-worldwide#how-to-get-to-the-reports-dashboard).
```

Comment:
The scrape interval _can_ be changed from what I see, and we do want it to be configurable in case a user wishes to collect more (or less) frequently. The 1hr is simply what we currently believe to be the best user experience compromise. I think just remove the bit that says that it's strict/cannot be changed, no need to add more context.